examples/dynamic-client/namespaced_custom_resource.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This example demonstrates the following:
+    - Creation of a namespaced scoped custom resource definition (CRD) using dynamic-client
+    - Creation of custom resources (CR) using the above created CRD
+    - List, patch(update), delete the custom resources
+    - Delete the custom resource defintion
+"""
+
+from kubernetes import config, dynamic
+from kubernetes.dynamic.exceptions import ResourceNotFoundError
+from kubernetes.client import api_client
+import time
+
+
+def main():
+    # Creating a dynamic client
+    client = dynamic.DynamicClient(
+        api_client.ApiClient(configuration=config.load_kube_config())
+    )
+
+    # fetching the custom resource definition (CRD) api
+    crd_api = client.resources.get(
+        api_version="apiextensions.k8s.io/v1beta1", kind="CustomResourceDefinition"
```

Comment:
please use v1 instead of v1beta1

Reply:
Thank you for pointing. I changed the CRD apiVersion to `apiextensions.k8s.io/v1`.